backend/views/admin.php

```diff
@@ -30,23 +32,24 @@
 			settings_fields( CF7_SMTP_TEXTDOMAIN . '-settings' );
 
 			/* This prints out the smtp settings */
-			echo '<div class="card smtp-settings-options">';
+			//echo '<div class="card smtp-settings-options">';
 			do_settings_sections( 'smtp-settings' );
 			submit_button();
-			echo '</div>';
-
+			//echo '</div>';
+			echo '<br>';
 			/* This prints the style options (template) */
-			echo '<div class="card smtp-style-options">';
+			//echo '<div class="card smtp-style-options">';
 			do_settings_sections( 'smtp-style' );
 			submit_button();
```

Comment:
I was thinking that now that it is no longer divided into cards all these keys no longer make sense. What do you think? is there still a need or can we get rid of them and leave just the one below?